content/disaster-recovery.md

```diff
@@ -0,0 +1,43 @@
+See [this](https://bosh.io/docs/director-configure-blobstore) to start configuring the director to use an external blobstore.
+
+## Steps for recovering a bosh director in the event of an AZ outage:
+
+### 1. Isolating the director
+To isolate the director in `zone_1` from the database to prevent it from modifying the state when the zone is up, it is important that:\
+a. The database passwords via terraform are rotated\
+b. All existing database connections are killed via psql
+### 2. Deploying another director
+There might be differences in deploying BOSH directors in another AZ based on whether we use the `create-env` approach or the `bosh deploy` approach.
+
+After isolating the director deployed by `create-env` in `zone_1` the process to deploy a second one in `zone_2` can be treated similar to a scratch installation. That means that the zone used in the director's deployment manifest to is changed to `zone_2` and the `bosh-state.json` is dropped before calling [create-env](https://bosh.io/docs/cli-v2/#create-env) and setting up the second director in `zone_2`.
+
+For directors deployed by `bosh deploy`, the deployment manifest should be updated the used zone in the manifest should be set to `zone_2`. Next,the number of [instances](https://bosh.io/docs/manifest-v2/#instance-groups) in the `instance_group` has to be increased to 2 and a second static ip (available in `zone_2`) should be configured.
+
+However, while redeploying thi director with the updated manifest, the director which this drector is the deployment of would
+first try to delete the old `bosh` in `zone_1`. This will most probably fail in case of a zone outage. This can be prevented by invoking [ignore](https://bosh.io/docs/cli-v2/#ignore) on the deployed `bosh`
+in `zone_1`.
+### 3. Disabling resurrection
+After `bosh` is successfully deployed in `zone_2` it will now take care for repairing its deployments. Most of them would
+be in error situation since they would have instances in the failing zone. The director would continuously trigger `scan_and_fix`
+tasks which would then also fail. 
+Therefore, an option would be to deploy the new directors with with disabled resurrection with other monitoring techniques in place.
```

Comment:
It is not completly clear what is meant by `...with other monitoring techniques in place.` We could leave this out or be more precise here.